src/SMESH.cpp

```diff
@@ -1132,7 +1132,7 @@ PYBIND11_MODULE(SMESH, mod) {
 	// C:\Users\Trevor\Work\Products\SMESH\install\include\smesh\SMESH_MeshVSLink.hxx
 	py::class_<SMESH_MeshVSLink, opencascade::handle<SMESH_MeshVSLink>, MeshVS_DataSource3D> cls_SMESH_MeshVSLink(mod, "SMESH_MeshVSLink", "None");
 	cls_SMESH_MeshVSLink.def(py::init<const SMESH_Mesh *>(), py::arg("aMesh"));
-	cls_SMESH_MeshVSLink.def(py::init<const SMESH_subMesh *>(), py::arg("aSubMesh"));
+	// FIXME cls_SMESH_MeshVSLink.def(py::init<const SMESH_subMesh *>(), py::arg("aSubMesh"));
```

Comment:
Does this not work for you? I frequently use viewing of sub-meshes so I'd prefer to keep this in there, at least for windows. Can we put it behind a ``#if defined(WIN32)`` if we must? Or does this work after you updated some of the SMESH issues?

Reply:
I can't remeber the problem with this. I will try to include the line next time.
Now back to windows, it seems like I introduced a regression to smesh:
```
   Creating library bin\SMDS.lib and object bin\SMDS.exp
SMDS.cpp.obj : error LNK2001: unresolved external symbol "public: static int SMDS_Mesh::chunkSize" (?chunkSize@SMDS_Mesh@@2HA)
bin\SMDS.pyd : fatal error LNK1120: 1 unresolved externals
```

Reply:
For Windows, use ``ENABLE_FORCE=ON``. There are a few link errors I get where it's referencing static variables in SMESH. They are initialized in the header and defined in the source which is perfectly valid C++, plus for some reason is causing these errors in the pybind11 wrapping. I need figure out if it's the way I'm wrapping the class or something wrong with pybind11 that needs a fix. Either way, use that CMake option.

Reply:
I tried again and everything seems to compile without error (linux). PR is updated.